libafl_frida/src/helper.rs

```diff
@@ -379,16 +377,6 @@ pub fn get_module_size(module_name: &str) -> usize {
     code_size
 }
 
-#[cfg(target_arch = "aarch64")]
-fn pc(context: &CpuContext) -> usize {
```

Comment:
What's wrong with these functions?

Reply:
They are no longer used in the code, so I deleted them.